packages/app-account/src/containers/ChainConnect.js

```diff
@@ -258,7 +258,7 @@ export class ChainConnect extends Component {
         <div className="ibweb-page app-auth">
           <Row>
             <Col {...colLayout}>
-              <img src={mainGraphic} alt="App info access" />
+              <img src={connectingHeader} alt="App info access" />
```

Comment:
This file needs to be refactored so that the `const someComponent = <SomeComponent />`  are in their own classes